lib/api/route/realm.g.dart

```diff
@@ -20,7 +20,7 @@ GetServerSettingsResult _$GetServerSettingsResultFromJson(
           .toList(),
       zulipFeatureLevel: (json['zulip_feature_level'] as num).toInt(),
       zulipVersion: json['zulip_version'] as String,
-      zulipMergeBase: json['zulip_merge_base'] as String?,
+      zulipMergeBase: json['zulip_merge_base'] as String,
```

Comment:
This doesn't seem to have a corresponding change in realm.dart.

Presumably this came about as you made some edits and missed rerunning `build_runner` for one of them, or something like that. I am puzzled that our CI didn't complain, though — `tools/check build_runner` should have failed because this doesn't match what a freshly-generated file should say. So that would be worth trying to debug.

Reply:
For future reference, this was fixed in #947.